spec/tictac_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Deyan', 'X') }
+  describe '#make_move' do
+    it 'check if it is correct' do
+      expect(new_game.make_move(player.team, 7)). to eql false
+    end
+    it 'Check if it is a string' do
+      expect(new_game.make_move(player.team, 'string')).to eql false
+    end
+    it 'check if the number is from 1 to 9' do
+      expect(new_game.make_move(player.team, 12)).to eql false
+    end
+  end
```

Comment:
As requested by the previous reviewer, please test for positive scenarios. For example, 

```ruby
it 'should validate move between 1 and 9' do
  expect(new_game.make_move(player.team, 8)).to eql true
end
```

The above is just an example. You are free to write your own version of positive scenario tests.